controllers/managedocs_controller_test.go

```diff
@@ -192,38 +192,49 @@ var _ = Describe("ManagedOCS controller", func() {
 			Namespace: testPrimaryNamespace,
 		},
 	}
-	pvc1StorageClassName := storageClassRbdName
-	pvc1Template := corev1.PersistentVolumeClaim{
+	pv1StorageClassName := storageClassRbdName
+	pv1Template := corev1.PersistentVolume{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "test-pvc-1",
+			Name:      "test-pv-1",
 			Namespace: testPrimaryNamespace,
 		},
-		Spec: corev1.PersistentVolumeClaimSpec{
-			StorageClassName: &pvc1StorageClassName,
+		Spec: corev1.PersistentVolumeSpec{
+			StorageClassName: pv1StorageClassName,
 			AccessModes: []corev1.PersistentVolumeAccessMode{
 				corev1.ReadWriteOnce,
 			},
-			Resources: corev1.ResourceRequirements{
-				Requests: corev1.ResourceList{
-					corev1.ResourceStorage: resource.MustParse("1Gi"),
+			Capacity: corev1.ResourceList{
+				corev1.ResourceStorage: resource.MustParse("1Gi"),
+			},
+			PersistentVolumeSource: corev1.PersistentVolumeSource{
+				RBD: &corev1.RBDPersistentVolumeSource{
+					CephMonitors: []string{
+						"0.0.0.0:6789",
+					},
+					RBDImage: "test",
```

Comment:
the source should be CSI, not RBD nor Cephfs.

Reply:
But this is just for our unit tests, we just need to make sure that the PVs are using OCS storage classes

Reply:
I agree with Dhruv on this one, our unit tests are mocking the stoagecluster and as such we have no notion of CSI.

Reply:
>But this is just for our unit tests, we just need to make sure that the PVs are using OCS storage classes

there is no relationship between the OCS storageclasses and the PersistentVolumeSource which is RBD and Cephfs mentioned here.

Reply:
Yes, I agree but we just want PVs using OCS storageClasses
It doesn't matter what PersistentVolumeSource we use